www/src/data/sidebars/doc-links.yaml

```diff
@@ -297,6 +297,8 @@
                   link: /docs/creating-a-transformer-plugin/
                 - title: Configuring Usage with Plugin Options
                   link: /docs/configuring-usage-with-plugin-options/
+                - title: Validating plugin options
```

Comment:
You'll need to update this in the mansion repo

Reply:
Wait, for real? Then we should delete this file from this repo. :facepalm:

Reply:
It's something that @jxnblk is aware of